contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,79 @@
+- You built a side project used by 1m people. Side projects are often great signs of crackedness/ownership, but they don’t have to be commercially successful - they just have to be interesting. 
+
+What’s cool about adopting this mentality is that it has noticeably increased the diversity of people applying to work with us at PostHog, as it ignores some of the traditional markers of status when it comes to job applications.
+
+### How to manage a cracked team
+
+A lot of traditional management behavior doesn’t apply here, and can actively slow down or demotivate your team.
+
+**Be cracked yourself**
+
+You need to demonstrate the behavior you want to see. Cracked people are extremely attuned to ‘do as I do’ not ‘do as I say’. This means spending a significant amount of time doing IC work. Do the stuff that arrogant senior people would find ‘boring’. Write tests. Respond to customer support queries. Go weirdly deep into some piece of analysis. Do something badly that’s not your job so you at least understand it better. 
+
+**Spend most of your time generating enthusiasm**
+
+It is unbelievably easy to find reasons not to do stuff at any kind of organization, including startups. Inertia is the default state of most businesses. Overcoming this inertia is mainly reliant on the enthusiasm of people to make new/cool things happen far more than their raw intelligence or competence. 
+
+This is why you should spend more time motivating people than trying to teach them new skills or making them do training - if they are cracked and staying motivated, they will train themselves. I don’t think I’ve yet read a blog post by someone I admire and learned from that says ‘I got good because I attended blah blah special leadership course on being great.’
+
+**Get out of the way**
+
+Cracked people are deeply intrinsically motivated. This means that extrinsic motivation, while still present in some form (we’ve all got to pay bills yeah), is way less important. Help cracked engineers take ownership of their own projects by encouraging them to make the decisions about what to do next. If they ask for help - ie. they actively want you to get in the way - ask questions about what they think should happen next, rather than giving them the answers. 
+
+Setting deadlines, creating arbitrary reporting obligations, or saying ‘if you ship X, you will get to Y title’ are examples of trying to use extrinsic motivation that you should avoid. 
+
+**Prioritize making cracked people more cracked, not making mediocre people good**
+
+Here is some subjective math:
+
+- The 20% of your team who are underperforming will take up 80% of your effort 
+- You may be able to get an underperformer’s output from 40% to 60% if you work hard with them
+- Doubling down on a cracked person’s performance can improve their output from 80% to 90%
+- It’s the 90% quality output that will delight your users, so you should focus on that, even if the absolute increase in performance is less 
```

Comment:
fly-by the switch from percentages being about amount of effort to quality score confused me here

would it be easier if the points on performance were "4/10" "6/10" 

i think that's what this means. it'll be a lot of work to improve an underperforming people from delivering mediocre to a slightly better mediocre, but way less effort to help a "cracked" engineer get from great to really great

??

(was reading this cos I kept hearing people say cracked and had no idea what was happening - thanks for writing it)

Reply:
Good point! Have updated.